tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/vimeo90k.py

```diff
@@ -50,3 +50,30 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
             annotations.append(SuperResolutionAnnotation(
                 MultiFramesInputIdentifier(list(range(len(input_data))), input_data), target))
         return ConverterReturn(annotations, None, None)
+
+class Vimeo90K_IFRNetDataset(BaseFormatConverter):
```

Comment:
I do not think that it is good idea to name dataset by model name as it does not contains any model specific logic. can it be named in more generic way? (potentially it can be reused for other models in future)

Reply:
Name has been changed to Vimeo90KIntermediateFrameInterpolationDataset